assets/src/modules/state/Layer.js

```diff
@@ -1165,10 +1166,15 @@ export class LayerGroupState extends LayerItemState {
      * Creating a layer group state
      * @param {LayerTreeGroupConfig} layerTreeGroupCfg - the layer item config
      * @param {number[]}             layersOrder       - the layers order
-     * @param {LayerGroupState}     [parentMapGroup]  - the parent layer group
+     * @param {boolean}              hideGroupCheckbox - the option instance from lizMap configuration
+     * @param {LayerGroupState}     [parentMapGroup]   - the hideGroupCheckbox option instance from lizMap configuration
      */
-    constructor(layerTreeGroupCfg, layersOrder, parentMapGroup) {
+    constructor(layerTreeGroupCfg, layersOrder, hideGroupCheckbox, parentMapGroup) {
         super('group', layerTreeGroupCfg, parentMapGroup);
+        // if checkboxes are hided for groups then they are checked by default
```

Comment:
```suggestion
        // if checkboxes are hidden for groups then they are checked by default
```
https://www.grammarflex.com/whats-the-past-tense-of-hide-is-it-hid-or-hidden/#Forms_of_the_verb_hide